operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -584,7 +584,11 @@ alertmanager:
     maxSkew: 1
     topologyKey: kubernetes.io/hostname
     whenUnsatisfiable: ScheduleAnyway
-
+    minDomains: 1
+    nodeAffinityPolicy: Honor
+    nodeTaintsPolicy: Honor
+    matchLabelKeys:
+      - pod-template-hash
```

Comment:
these were added in k8s 1.26 and 1.27. The mimir-distributed helm chart supports down to 1.20 at the moment. Let's also have the commented so that they don't break existing installations on <1.27

this comment really applies to all instances of TSC in values.yaml